deploy-to-azure/azext_deploy_to_azure/dev/resources/resourcefiles.py

```diff
@@ -74,12 +74,17 @@
             creds: ${{ SECRETS.AZURE_CREDENTIALS }}
 
         - name: 'Deploy to Azure Container Instances'
-          uses: azure/CLI@v1
+          uses: azure/aci-deploy@v1
           with:
-            azcliversion: 2.0.77
-            inlineScript: |
-              az container create --resource-group resource_name_place_holder --name app_name_place_holder --image container_registry_name_place_holder.azurecr.io/app_name_place_holder:${{ github.sha }} --ports 80 port_number_place_holder --dns-name-label app_name_place_holder --registry-username ${{ SECRETS.REGISTRY_USERNAME }} --registry-password ${{ SECRETS.REGISTRY_PASSWORD }}
-"""
+            resource-group resource_name_place_holder
```

Comment:
@mitsha-microsoft  , **resource-group:** . You are missing ':'
Did this workflow run successfully for you?

Reply:
Fixed these typos and the workflows run successfully (even with custom ports)